### PR TITLE
Feature/cplp 1295 select list

### PR DESCRIPTION
**Pull request by PI-FerasAhmad**
- Multi select list improved: add events onChange
- Select List component added

### CODE REVIEW

File: cx-portal-shared-components/src/components/basic/MultiSelectList/Components/SelectOptions.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      }}
+    >
+      <Box
+        sx={{
+          borderBottom: '1px solid #f2f2f2 !important',
+          width: '100%',
+          paddingBottom: '10px',
+        }}
+      >
+        {parts.map((part: any, index: any) => (
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/MultiSelectList/Components/SelectOptions.tsx:27]: at least index should be of type `number`

**Review by oyo — APPROVED** [review]
generally approved - sometimes it's difficult to find anything better than `any`...

File: cx-portal-shared-components/src/components/basic/MultiSelectList/Components/SelectOptions.tsx (hunk 1 above)
Comment (PI-FerasAhmad) [reply cx-portal-shared-components/src/components/basic/MultiSelectList/Components/SelectOptions.tsx:27]: you right, I will change them ;-)

Comment (PI-FerasAhmad) [reply cx-portal-shared-components/src/components/basic/MultiSelectList/Components/SelectOptions.tsx]: @oyo I just add type for parts, can you please take a look? Thank you :-)

**Review by oyo — APPROVED** [review]
looks even better now ;)

**Pull request merged**